### PR TITLE
LND: update to version 0.14.1

### PR DESCRIPTION
**Pull request by Stadicus**
This change updates LND to the latest official release, known to contain 
improvements due to breaking changes in v0.14.0

### CODE REVIEW

File: lnd.md [hunk 1]
```diff
@@ -32,48 +32,47 @@ We'll download, verify and install LND.
...
 
-* Get the public signing keys of Roasbeef and add it to our GPG keyring
+* You should already have the public key from LND's developer [Oliver Gugger](https://github.com/guggero){:target="_blank"} from the Bitcoin Core installation.
+  If not, you can get it now and add it to our GPG keyring
 
   ```sh
-  $ curl https://keybase.io/roasbeef/pgp_keys.asc | gpg --import
+  $ curl https://keybase.io/guggero/pgp_keys.asc | gpg --import
+  > ...
+  > gpg: key 0x8E4256593F177720: public key "Oliver Gugger <gugger@gmail.com>" imported
```
Comment (sig-noi) [inline lnd.md:46]: I get the following output when checking against pre-existing keys. I think this is OK but might include some of the odd detail here if expected so as to reassure the user.

gpg: key 8E4256593F177720: 1 signature not checked due to a missing key
gpg: key 8E4256593F177720: "Oliver Gugger <gugger@gmail.com>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1

Or in the case of a new keyring

gpg: key 8E4256593F177720: 1 signature not checked due to a missing key
gpg: /home/lnd/.gnupg/trustdb.gpg: trustdb created
gpg: key 8E4256593F177720: public key "Oliver Gugger <gugger@gmail.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: no ultimately trusted keys found

Comment (Stadicus) [reply lnd.md:46]: Yes, that's exactly what's expected if you already have the key in your keyring:
* Processed: 1
* Unchanged: 1
Thanks for checking!

**Review by sig-noi — APPROVED** [review]
Line 218 has another ‘still with user “bitcoin”’ but should be with user “lnd” correct? Bitcoin can’t run lnd when I try. Additionally I wasn't bable to get lncli to create or open a wallet (can't recall the error and not by computer for the next few days). Unsure if there are issues left to fix but would not be able to fully review for a few days.

**Review by VajraOfIndra — APPROVED** [review]
LGTM!

Part 1: Installation of v0.14.1: tACK
Part 2: LND user and RPC credentials: utACK

**Comment by Stadicus** [comment]
Thanks @VajraOfIndra. I fixed a few more `"bitcoin"` user instances and replaced them with `"lnd"`.

PTAL (Please take another look :)

**Pull request merged**